library/src/main/java/com/okta/oidc/AuthorizationStatus.java

```diff
@@ -31,5 +31,31 @@
     /**
      * Signed out authorization status. Browser session is cleared.
      */
-    SIGNED_OUT
+    SIGNED_OUT,
+
+    /**
+     * Operation was canceled.
+     */
+    CANCELED,
+
+    /**
+     * Operation resulted in an exception.
+     */
+    ERROR,
+
+    /**
+     * Email verified and user is authenticated with a valid browser session. Client should handle
+     * this status by calling
+     * {@link com.okta.oidc.clients.web.WebAuthClient#signIn(Activity, AuthenticationPayload)}
+     * again.
+     */
+    EMAIL_VERIFICATION_AUTHENTICATED,
```

Comment:
I'm confused by this state. So if the email is verified and the user is already authenticated why do we need to sign in again? Aren't we already signed in?

Reply:
Sort of. The user is authenticated, the browser have a session but the PKCE flow isn't completed so they are authenticated but not authorized. Developers have to call sign-in again to finish the PKCE flow. But the user doesn't have to enter a user/pw since they already have a session (AUTHENTICATED). If they are UNAUTHENTICATED that means the email is verified but they have to enter credentials.

Reply:
Might be helpful to include some of that color in the comments. I doubt I'd be the only person confused.